bin/local-env/install-wordpress.sh

```diff
@@ -92,9 +92,9 @@ fi
 # If the 'wordpress' volume wasn't during the down/up earlier, but the post port has changed, we need to update it.
 echo -e $(status_message "Checking the site's url...")
 CURRENT_URL=$(wp option get siteurl)
-if [ "$CURRENT_URL" != "http://localhost:$HOST_PORT" ]; then
-	wp option update home "http://localhost:$HOST_PORT" --quiet
-	wp option update siteurl "http://localhost:$HOST_PORT" --quiet
+if [ "$CURRENT_URL" != "http://127.0.0.1:$HOST_PORT" ]; then
```

Comment:
Reason for updating `localhost` with `127.0.0.1` - https://github.com/node-fetch/node-fetch/issues/1624